Dockerfile

```
@@ -1,11 +1,10 @@
 # Docker 17.05 or higher required for multi-stage builds
-FROM rust:1.48 as builder
+FROM rust:1.68 as builder
```

Apparently `rust:1.68` is now based on `bullseye`. Let's specify the same debian release in both the builder and the final image so they don't go out of sync.

```suggestion
FROM rust:1.68-bullseye as builder
```
(or `buster`)

---

Going with Buster for now.